scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/ServerHandshaker.java

```diff
@@ -96,6 +96,9 @@
 	 */
 	private DTLSFlight lastFlight;
 
+	/** Is the client wanted to authenticate itself? */
```

Comment:
I didn't get the difference. 
FMPOV, "is the client wanted" = "someone want the client to"

Reply:
Ok I think it was wording missunderstanding.

I understand your sentence as :  do the client want to authenticate ? 
So I checked "Is the client wanted to authenticate itself?" with google translate which translate it in the way I understand it.

If we use "requested" instead of "wanted", I'm less confused and google translate too. But maybe your formulation is OK... we need a fluent in English man ! (maybe you are :p)